flang/test/Lower/intrinsic-procedures/command_argument_count.f90

```diff
@@ -0,0 +1,10 @@
+! RUN: flang-new -fc1 -emit-fir %s -o - | FileCheck %s
+! RUN: flang-new -fc1 -fdefault-integer-8 -emit-fir %s -o - | FileCheck --check-prefixes=CHECK,CHECK-64  %s
+
```

Comment:
Could you add a comment explaining the rationale behind using `flang-new` here? I'm happy with the change, but it might be useful to document this for our future self.

Reply:
Done. Also changed default flang-new usage to use bbc to keep consistency with other tests.